rust/src/cfd.rs

```diff
@@ -183,10 +197,11 @@ pub async fn settle(order: Order, offer: Offer) -> Result<()> {
 
     let channel_manager = wallet::get_channel_manager()?;
 
-    let custom_output_id = *channel_manager
-        .custom_outputs()
-        .first()
-        .context("No custom outputs in channel")?;
+    let custom_output_id = base64::decode(cfd.custom_output_id)?;
```

Comment:
🛑 wait, why does this work? 

The `custom_output_id` is randomly generated from the library.

Reply:
I think we are just reloading it from the database, where it is serialised as base64.

Reply:
oh, right, I misread it then :)